lib/ansible/modules/cloud/openstack/os_security_group.py

```diff
@@ -126,6 +135,9 @@ def main():
         if state == 'present':
             if not secgroup:
                 secgroup = cloud.create_security_group(name, description)
+                if egress_default_rules == 'absent':
```

Comment:
you're right, I didn't think about others default rules. I fixe the patch. Thanks for the review